README.md

```diff
@@ -33,15 +33,77 @@ e.g: (for script named `script.py`)
 1. If `log_file_prefix=prefix` --> log file will be stored at `./prefix_wesci_log`
 1. If `log_file_prefix=/log/prefix` --> log file will be stored at `/log/prefix_wesci_log`
 
-### Adding Input/Output Parameters and Files
+### Logging Params
+**Note:** We don't support all variable types yet. When an unsupported type is passed as input/output param, a warning will be issued when the code is executed, and the param will be ignored.
+For all supported params types - you'll be able to click on their row in the params table at [app.we-sci.com](https://app.we-sci.com) and see their preview in the bottom preview box.
+
+The logger supports multiple methods for logging your script's params, trading-off between the amount of code needed to start logging and accuracy of logging:
+
+#### Automatically Logging All Global Params
+```python
+### Your script's parameters and calculations ###
+# ...
+# ...
+# ...
+logger.log(globals())
+```
+The logger can accept the `globals()` dictionary (or any other dictionary) and log the parameters provided as **output params**.
```

Comment:
**Curious:** What does "or any other dictionary" actually mean ?

Reply:
you can pass is whatever you want. We recommend passing in `globals()` for the best effect. But you can pass in a manually constructed dict

Reply:
Is it "self explanatory" to a Python programmer ? or do you think it is worth an example ?

Reply:
there is an example - with globals(). It's a bad idea IMHO to provide an example of something we don't want them to do